### ISSUE TITLE
Table immediately crashes on launch with Script Runtime Error: Runtime error - Description unavailable

### ISSUE THREAD

**Issue by KrakenByteKraken** [vpinball/vpinball#2033]
**This issue only occurs in some tables** but, when one of these tables start, the table immediately crashes on launch returning a "**Runtime error - Description unavailable**":

![Description unavailable](https://github.com/user-attachments/assets/d7b7dc09-4e26-4aae-bc7b-5fc6c0e7f9a1)

The issue doesn't appear when using **RC5** (**VPinballX_GL-10.8.0-2015-6f63503-Release-win-x64.zip**): **The Release Candidate 5 works fine**.

**But newest releases crash**, for example, you can reproduce this problem with the last **Master** (**VPinballX_GL-10.8.0-2035-8405e87-Release-win-x64.zip**) or even the **10.8.1** branch (**VPinballX_BGFX-10.8.1-2737-eaad41d-Release-win-x64.zip**).

I found only a few tables with this issue, but I don't remember all. For example, you can reproduce this problem with the table [Indiana Jones The Pinball Adventure (Williams 1993) VPWmod](https://vpuniverse.com/files/file/8568-indiana-jones-the-pinball-adventure-williams-1993-vpwmod/).

**Comment by francisdb** [comment]
Is this screenshot from Indiana Jones 1.1?

**Comment by KrakenByteKraken** [comment]
Yes, it is: The example screenshot is also from the example table: Indiana Jones Pinball Adventure.

**Comment by francisdb** [comment]
![image](https://github.com/user-attachments/assets/9b794a5c-c5f6-4311-89a6-066b873721e2)

Somehow checking if the `user` folder exists fails on your system
https://github.com/vpinball/vpinball/blob/8405e8709cebad7ca3be5b3e699ec421e4e24360/txt/CommandReference.txt#L227

The logic that gets that `UserDirectory` is here

https://github.com/vpinball/vpinball/blob/8405e8709cebad7ca3be5b3e699ec421e4e24360/src/parts/pintable.cpp#L461-L473

Maybe it is missing on your system?

**Comment by KrakenByteKraken** [comment]
OK, that started to fixed it, thank you.

Following your instructions, I made a new folder called "user" where the main VPX EXE is stored (e.g. C:\Visual Pinball\user) and it made the trick.

After a little testing, I see that now it works in all 10.8.0 versions (RC5 which was working from start and never crashed, but also with the last). It works also in 10.8.1 version in BGFX flavor, but doesn't work in GL flavor. It's a strange and irregular behavior.

For that, I wonder:

- Why did it work when I played with RC5 if this folder didn't exist then?
- Why is not included in the zip pack like all other folders like "docs", "plugins", "scripts", "tables", etc.?
- Why not to use the "%appdata%\Roaming\VPinballX" folder for that?

Anyway, thank you very much for your help.

**Comment by francisdb** [comment]
And can we add a message to that error that points the user to the solution?
